lang/rust-bootstrap/Portfile

```diff
@@ -7,8 +7,8 @@ PortGroup   rust            1.0
 name                        rust-bootstrap
 
 # keep in mind that you also need to update cargo.crates at the end of this file
-set version_current         1.71.1
-set version_m1              1.70.0
```

Comment:
Mentioned on the Trac ticket, but I think we should probably target 1.73.0 *before* 1.74.1 or 1.75.0 so we can better support older versions of macOS before we have to restrict to 10.12 or later because of Rust build system changes.

Reply:
Ideally, It would be best to build the whole chain, i.e.

```
rust-bootstrap         rust    
1.71.0                 1.72.1
1.72.0                 1.73.0
1.73.0                 1.74.1
1.74.0                 1.75.0
```

to see where things break for older OS versions.